src/App/Components/Profile/BookMark.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import _ from 'lodash';
+import ReactImageFallback from 'react-image-fallback';
+import propTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+export default function BookMark(props) {
+  const { data, remove, owner } = props;
+
+  return !_.isEmpty(data) ? (
+    <div>
+      {data.bookmarks.map(bookmark => (
+        <div key={bookmark.Article.id} className="row mt-4 bookmark_container p-3">
```

Comment:
Same here @alainburindi

Reply:
There are many iterations of this actually.

Reply:
@nkpremices can you sync with @alainburindi and show him an example of what you mean?

Reply:
I explained to @nkpremices the reason and I have destructured where it was possible